test/units/path_test.rb

```diff
@@ -48,6 +48,13 @@ def test_path_with_multi_value_param_safely_encoded
     assert_equal url, path.markalign('middle', 'center').to_url
   end
 
+  def test_path_params_with_spaces
+    url = 'https://demo.imgix.net/images/demo.png?txt=Hello%20World&s=c4d702123e15b88495be582c133cc9ed'
+    path = client.path('/images/demo.png')
+
+    assert_equal url, path.to_url(txt: 'Hello World')
```

Comment:
Switching line 52 to `url = 'https://demo.imgix.net/images/demo.png?txt=Hello%20World&s=664b2620c89d53442139df0d2a36be9c'` should do the trick. The signature just needed to be changed to match the output URL. I generated the new signature by running `md5 -s '10adc394/images/demo.png?txt=Hello%20World'` in my terminal (MD5 of the test signing key + path).